File: 2025_11/leetcode2025-11-22.py
from typing import List

class Solution:
    def minimumOperations(self, nums: List[int]) -> int:
        return sum(min(i%3, 3-i%3) for i in nums)

if __name__ == "__main__":
    s = Solution()
    nums = [1,2,3,4]
    print(s.minimumOperations(nums))

File: leetcode2025/2025_11/leetcode2025-11-22.cpp
#include <iostream>
#include <vector>
using namespace std;

class Solution {
public:
    int minimumOperations(vector<int>& nums) {
        int res = 0;
        for (int i : nums){
            res += min(3-i%3, i%3);
        }
        return res;
    }
};

int main(){
    Solution s;
    vector<int> nums = {1,2,3,4};
    cout << s.minimumOperations(nums) << endl;
}
